README.md

```diff
@@ -4,6 +4,7 @@
 <!-- badges: start -->
 [![CRAN](https://www.r-pkg.org/badges/version/rhino)](https://cran.r-project.org/package=rhino)
 [![CI](https://github.com/Appsilon/rhino/actions/workflows/ci.yml/badge.svg)](https://github.com/Appsilon/rhino/actions/workflows/ci.yml)
+[![Rhino Test](https://github.com/Appsilon/rhino/actions/workflows/rhino-test.yml/badge.svg?branch=bot%2Fapp-push-test)](https://github.com/Appsilon/rhino/actions/workflows/rhino-test.yml)
```

Comment:
1. I feel that the name displayed on the badge is misleading.
2. I am also not sure if the badge should be here - e.g. we don't have a badge for e2e tests, which are IMO more important.

Reply:
Turns out you cannot use custom names for GitHub Actions badges :shrug: We'd need to edit the actual names in the workflow files and I feel it's too much for this PR. I'll leave the README as is for now (just the CI badge) and save idea for another time.